code/unclassified/src/magic_square/magic_square.c

```diff
@@ -57,6 +51,6 @@ void MagicSquare(int n)
 int main()
 {
     int n = 7; // Works only when n is odd
-    MagicSquare(n);
-    return 0;
+    magic_square(n);
```

Comment:
Any way you could modify this to return an array? Also, are you sure C style guide says underscore_case over camelCase?

Reply:
Yes, see the [style guide](https://github.com/christianbender/cosmos/tree/master/guides/coding_style/c#indentation)